tests/test_scooby.py

```diff
@@ -87,7 +87,7 @@ def test_get_version():
     assert version == numpy.__version__
     assert name == "numpy"
     name, version = scooby.get_version("no_version")
-    assert version == scooby.report.VERSION_NOT_FOUND
+    assert version == "0.1.0"  # Assuming this package never updates
```

Comment:
@prisae, I assume we won't ever update this package?

The changes in this PR make it tough to not get a version for a package `pkg_resources` will get the distribution version which has to be set afaik

Reply:
I am absolutely not sure I like this. So that package has no version. That is the whole point of that package. So scooby should not print "0.1.0", because it is simply wrong.

Reply:
OK, pip does assign it by default 0.1.0. That is nothing I did, so I would not "update" the version number, it does not has one. Pip gives it.

If the package would not be installed, but in the python path, then scooby would still return `Version unknown` (hence `scooby.report.VERSION_NOT_FOUND`), so I think we should test that.